### DISCUSSION TITLE
Multiple Scalebar

### DISCUSSION THREAD

**Discussion by robbaker292** [Mapsui/Mapsui#1403] [Q&A] [answered]
As there appears to be no documentation on this, how does one go about using multiple scalebars with different values:

```
map.Widgets.Add(new Mapsui.Widgets.ScaleBar.ScaleBarWidget(map)
            {
                TextAlignment = Alignment.Center,
                HorizontalAlignment = HorizontalAlignment.Left,
                VerticalAlignment = VerticalAlignment.Top,
                ScaleBarMode = Mapsui.Widgets.ScaleBar.ScaleBarMode.Both
            });
```
Seems to work for one, but it's not clear how to use the `UnitConverters` to specify miles rather than kilometers. Equally, there appears to be nothing about what `ScaleBarMode` does - any ideas?

**Comment by charlenni** [comment, marked as answer]
Did you look into the sample for [ScaleBars](https://github.com/Mapsui/Mapsui/blob/master/Samples/Mapsui.Samples.Common/Maps/Widgets/ScaleBarSample.cs)?

**Reply by robbaker292** [reply]
Thanks, that page never came up in my Googling!
